driver/src/main.rs

```diff
@@ -97,11 +41,13 @@ fn main() {
 		let received = rx.recv().unwrap();
 		println!(": {}", received);
 
-		// initializing all needed web3 variables
 		let db_host = env::var("DB_HOST").unwrap();
 		let db_port = env::var("DB_PORT").unwrap();
-		let client = Client::connect(&db_host, db_port.parse::<u16>().unwrap())
-			.expect("Failed to initialize standalone client");
+		let db_instance =
+			db_interface::DbMongoInstance::new(db_host, db_port).unwrap_or_else(|err| {
+				println!("Problem creating DbInterface: {}", err);
+				process::exit(1);
```

Comment:
🙅‍♂️ please no process::exit

Reply:
This will be refactored in the other PR...

Reply:
It would then be nice to not have this change at all in this PR. But not a big deal.